cmd/kubeadm/app/cmd/upgrade/diff.go

```diff
@@ -88,14 +88,6 @@ func newCmdDiff(out io.Writer) *cobra.Command {
 
 	options.AddKubeConfigFlag(cmd.Flags(), &flags.kubeConfigPath)
 	options.AddConfigFlag(cmd.Flags(), &flags.cfgPath)
-	cmd.Flags().StringVar(&flags.apiServerManifestPath, "api-server-manifest", defaultAPIServerManifestPath, "path to API server manifest")
-	cmd.Flags().MarkDeprecated("api-server-manifest", "This flag is deprecated and will be removed in a future release.")
-	cmd.Flags().StringVar(&flags.controllerManagerManifestPath, "controller-manager-manifest", defaultControllerManagerManifestPath, "path to controller manifest")
-	cmd.Flags().MarkDeprecated("controller-manager-manifest", "This flag is deprecated and will be removed in a future release.")
-	cmd.Flags().StringVar(&flags.schedulerManifestPath, "scheduler-manifest", defaultSchedulerManifestPath, "path to scheduler manifest")
-	cmd.Flags().MarkDeprecated("scheduler-manifest", "This flag is deprecated and will be removed in a future release.")
-	cmd.Flags().IntVarP(&flags.contextLines, "context-lines", "c", 3, "How many lines of context in the diff")
```

Comment:
https://github.com/kubernetes/kubernetes/blob/bce499c13661a9b6c376caa53655a132d903d79b/cmd/kubeadm/app/cmd/upgrade/diff.go#L57-L61 

As the CI failure, those variables can be removed as well.

Overall LGTM.

Reply:
looks like i also removed context-lines which should not be done.

Reply:
updated.